jobs/telemetry_batch_view.py

```diff
@@ -40,16 +37,14 @@ def submit_job():
         "--master", "yarn",
         "--deploy-mode", "client",
         "--class", environ["TBV_CLASS"],
-        "target/scala-2.11/telemetry-batch-view-1.1.jar",
+        artifact_file,
     ])
+
     call_exit_errors(command)
 
 
 if environ.get("DO_ASSEMBLY", "True") == "True":
```

Comment:
If the plan is to move away from build and compile completely, `DO_ASSEMBLY` should probably be removed/renamed.